website/django/data/models.py

```diff
@@ -478,3 +478,101 @@ class CurrentVariant(models.Model):
     class Meta:
         db_table = 'currentvariant'
         managed = False
+
+
+class InSilicoPriors(models.Model):
+    Variant = models.OneToOneField('Variant', primary_key=True)
+
+    varType = models.TextField()
+    varLoc = models.TextField()
+    applicablePrior = models.TextField(null=True)
```

Comment:
previously we've avoided using null=True on text fields per recommendation from django docs https://docs.djangoproject.com/en/dev/ref/models/fields/#null

I'm open to being convinced otherwise though if there's a good reason for it :)

Reply:
Sounds good to me; I agree that we should avoid having multiple values that represent null.  (I will say the nice thing about using null to represent an empty value is that it's available irrespective of type: any column type can all be nullable, whereas a special string value for null strings means you need to treat other possibly empty types differently.)

Anyway, Should those fields instead have `default=''` or `default='-'` specified, then? I've noticed that both are used to represent empty values and wasn't sure which one is preferred (I'd personally prefer '' if we're going to use a string).